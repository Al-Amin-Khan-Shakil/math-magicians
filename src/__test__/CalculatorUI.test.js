import renderer from 'react-test-renderer';
import CalculatorButtons from '../components/CalculatorUI';

it('Test the calculator ui component', () => {
  const tree = renderer.create(<CalculatorButtons />).toJSON();
  expect(tree).toMatchSnapshot();
});
