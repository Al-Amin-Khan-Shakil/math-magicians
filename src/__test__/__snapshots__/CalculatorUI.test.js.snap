// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the calculator ui component 1`] = `
<div
  className="cal-wrapper"
>
  <h2
    className="cal-title"
  >
    Let's do some math!
  </h2>
  <div
    className="main-container"
  >
    <div
      className="cal-container"
    >
      <div
        className="output"
      >
        <div
          className="display"
        >
          0
           
           
        </div>
      </div>
      <button
        className="keypad"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        AC
      </button>
      <button
        className="keypad"
        onClick={[Function]}
        type="button"
        value="+/-"
      >
        +/-
      </button>
      <button
        className="keypad"
        onClick={[Function]}
        type="button"
        value="%"
      >
        %
      </button>
      <button
        className="keypad last-col"
        onClick={[Function]}
        type="button"
        value="รท"
      >
        รท
      </button>
      <button
        className="keypad"
        onClick={[Function]}
        type="button"
        value="7"
      >
        7
      </button>
      <button
        className="keypad"
        onClick={[Function]}
        type="button"
        value="8"
      >
        8
      </button>
      <button
        className="keypad"
        onClick={[Function]}
        type="button"
        value="9"
      >
        9
      </button>
      <button
        className="keypad last-col"
        onClick={[Function]}
        type="button"
        value="x"
      >
        x
      </button>
      <button
        className="keypad"
        onClick={[Function]}
        type="button"
        value="4"
      >
        4
      </button>
      <button
        className="keypad"
        onClick={[Function]}
        type="button"
        value="5"
      >
        5
      </button>
      <button
        className="keypad"
        onClick={[Function]}
        type="button"
        value="6"
      >
        6
      </button>
      <button
        className="keypad last-col"
        onClick={[Function]}
        type="button"
        value="-"
      >
        -
      </button>
      <button
        className="keypad"
        onClick={[Function]}
        type="button"
        value="1"
      >
        1
      </button>
      <button
        className="keypad"
        onClick={[Function]}
        type="button"
        value="2"
      >
        2
      </button>
      <button
        className="keypad"
        onClick={[Function]}
        type="button"
        value="3"
      >
        3
      </button>
      <button
        className="keypad last-col"
        onClick={[Function]}
        type="button"
        value="+"
      >
        +
      </button>
      <button
        className="keypad num-zero"
        onClick={[Function]}
        type="button"
        value="0"
      >
        0
      </button>
      <button
        className="keypad"
        onClick={[Function]}
        type="button"
        value="."
      >
        .
      </button>
      <button
        className="keypad last-col"
        onClick={[Function]}
        type="button"
        value="="
      >
        =
      </button>
    </div>
  </div>
</div>
`;
